cores/esp32/IPAddress.cpp

```diff
@@ -395,6 +395,9 @@ IPAddress &IPAddress::from_ip_addr_t(const ip_addr_t *addr) {
 #endif /* LWIP_IPV6_SCOPES */
   } else {
     _type = IPv4;
+    _address.dword[0] = 0;
+    _address.dword[1] = 0;
+    _address.dword[2] = 0;
```

Comment:
It's better to fill the buffer to zeroes, in case that the index changes in the future
```suggestion
    memset(_address.bytes, 0, sizeof(_address.bytes));
```

Reply:
Resolved :-)